components/table/TableRow.js

```diff
@@ -62,14 +75,26 @@ const factory = (Checkbox) => {
       const inputType = utils.inputTypeForPrototype(this.props.model[key].type);
       const inputValue = utils.prepareValueForInput(value, inputType);
       const checked = inputType === 'checkbox' && value ? true : null;
-      return (
-        <input
-          checked={checked}
-          onChange={this.handleInputChange.bind(null, index, key, inputType)}
-          type={inputType}
-          value={inputValue}
-        />
-      );
+
+      if (inputType === 'text') {
+        return (
+          <div
+            contentEditable
+            suppressContentEditableWarning
+            onInput={this.handleInputChange.bind(null, index, key, inputType)}>
+            {inputValue}
+          </div>
+        );
+      } else {
```

Comment:
I dont think the else > return is necessary in this case. See: http://eslint.org/docs/rules/no-else-return